tests/FieldMessageTests.cpp

```diff
@@ -79,3 +79,10 @@ TEST(MessageModificationTests, AddInvalidParameterTypeTest) {
     EXPECT_ANY_THROW(message.SetIntField(Field::PlayerName, 93));
     EXPECT_ANY_THROW(message.SetStringField(Field::TankHp, "30"));
 }
+
+TEST(MessageModificationTests, UniqueIdsTest) {
```

Comment:
`TEST`, `MessageModificationTESTs`, `UniqueIdTEST`. So many `TEST`s!
One `TEST` macro is enough for `TEST(Message, UniqueId)`

`tests/FieldMessageTests.cpp` also can be renamed to `tests/FieldMessage.cpp`